### DISCUSSION TITLE
Toggle button instead of self-referential pronouns

### DISCUSSION THREAD

**Discussion by Drakemoor** [brianpetro/obsidian-smart-connections#450] [Ideas]
Currently, to trigger a lookup on my notes, I have to use self-referential pronouns like: my, l, me, mine, our, ours, us, we

Instead, I would like to have a toggle button above the Chat so I can easily switch between my notes and general talk with the AI. 

Thank you for your consideration. 🙇🏻‍♂️

**Comment by brianpetro** [comment]
Good idea. Moving further discussion here https://github.com/brianpetro/obsidian-smart-connections/issues/451#issue-2116868397
